### PR TITLE
Changing travis profile to coveralls

### PR DESCRIPTION
**Pull request by abhimanyugupta07**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.4.0] - 2020-08-17
```
Comment (massdosage) [inline CHANGELOG.md]: Needs to go to 2.0.0 since this is a user facing, non-backward compatible change.

Comment (abhimanyugupta07) [inline CHANGELOG.md]:
```suggestion
## [2.0.0] - 2020-08-17
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.0.0] - 2020-08-17
+### Changed
+- renamed `travis` profile to `coveralls`.
+
```
Comment (abhimanyugupta07) [inline CHANGELOG.md:10]:
```suggestion
```

**Review by massdosage — APPROVED** [review]
Remember to set 2.0.0 when you do the release! (or update the pom.xml now).

**Pull request merged**